components/Buttons/AddButton.js

```diff
@@ -0,0 +1,24 @@
+import StyledButton from "./StyledButton";
+import SVGIcon from "../Icons/SVGIcon";
+
+export default function AddButton({ isAddMode, onClick }) {
+  return (
+    <>
+      <StyledButton
+        type="button"
+        variant={isAddMode ? "hide" : "show"}
+        onClick={onClick}
+      >
+        {isAddMode ? "Close" : "Add"}
+        {!isAddMode && (
+          <SVGIcon
+            variant="plus"
+            width="2rem"
+            color="#04BF45"
+            aria-label="plus"
+          />
+        )}
+      </StyledButton>
```

Comment:
This is a nice component, but AddButton, SearchButton and TranslateButton are very similar. At some point you might think about generalizing these tree buttons into your StyledButton component and pass down the text and Icons as children (you can always use the prop children to receive the nested elements of a component). But for now its fine!

Reply:
Thank you! You are right, good point! 
I'll try to generalize them later on.